Failed to merge upstream when running unit tests for a non-member

https://github.com/knative/serving/pull/12036/checks?check_run_id=3795128252

raw logs
```
2021-10-04T18:51:44.9146235Z Found online and idle hosted runner in the current repository's organization account that matches the required labels: 'ubuntu-latest'
2021-10-04T18:51:44.9882233Z Waiting for a Hosted runner in the 'organization' to pick this job...
2021-10-04T18:51:45.0895482Z Job is waiting for a hosted runner to come online.
2021-10-04T18:51:48.3644622Z Job is about to start running on the hosted runner: Hosted Agent (hosted)
2021-10-04T18:51:52.0198367Z Current runner version: '2.283.1'
2021-10-04T18:51:52.0221947Z ##[group]Operating System
2021-10-04T18:51:52.0222778Z Ubuntu
2021-10-04T18:51:52.0223218Z 20.04.3
2021-10-04T18:51:52.0223578Z LTS
2021-10-04T18:51:52.0224010Z ##[endgroup]
2021-10-04T18:51:52.0224481Z ##[group]Virtual Environment
2021-10-04T18:51:52.0225057Z Environment: ubuntu-20.04
2021-10-04T18:51:52.0225530Z Version: 20210929.1
2021-10-04T18:51:52.0226401Z Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20210929.1/images/linux/Ubuntu2004-README.md
2021-10-04T18:51:52.0227582Z Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210929.1
2021-10-04T18:51:52.0228303Z ##[endgroup]
2021-10-04T18:51:52.0228846Z ##[group]Virtual Environment Provisioner
2021-10-04T18:51:52.0229397Z 1.0.0.0-master-20210914-1
2021-10-04T18:51:52.0229883Z ##[endgroup]
2021-10-04T18:51:52.0231550Z ##[group]GITHUB_TOKEN Permissions
2021-10-04T18:51:52.0232709Z Actions: read
2021-10-04T18:51:52.0233210Z Checks: read
2021-10-04T18:51:52.0233677Z Contents: read
2021-10-04T18:51:52.0234116Z Deployments: read
2021-10-04T18:51:52.0234611Z Discussions: read
2021-10-04T18:51:52.0235193Z Issues: read
2021-10-04T18:51:52.0235688Z Metadata: read
2021-10-04T18:51:52.0236172Z Packages: read
2021-10-04T18:51:52.0236588Z Pages: read
2021-10-04T18:51:52.0237067Z PullRequests: read
2021-10-04T18:51:52.0237580Z RepositoryProjects: read
2021-10-04T18:51:52.0238164Z SecurityEvents: read
2021-10-04T18:51:52.0238628Z Statuses: read
2021-10-04T18:51:52.0239186Z ##[endgroup]
2021-10-04T18:51:52.0241915Z Prepare workflow directory
2021-10-04T18:51:52.0755303Z Prepare all required actions
2021-10-04T18:51:52.0763681Z Getting action download info
2021-10-04T18:51:52.2917708Z Download action repository 'actions/setup-go@v2' (SHA:331ce1d993939866bb63c32c6cbbfd48fa76fc57)
2021-10-04T18:51:54.2570978Z Download action repository 'actions/checkout@v2' (SHA:5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f)
2021-10-04T18:51:54.4036329Z Download action repository 'andstor/file-existence-action@v1' (SHA:87d74d4732ddb824259d80c8a508c0124bf1c673)
2021-10-04T18:51:54.6109643Z Download action repository 'codecov/codecov-action@v1' (SHA:29386c70ef20e286228c72b668a06fd0e8399192)
2021-10-04T18:51:54.8992830Z ##[group]Run actions/setup-go@v2
2021-10-04T18:51:54.8993323Z with:
2021-10-04T18:51:54.8993628Z   go-version: 1.16.x
2021-10-04T18:51:54.8993941Z   stable: true
2021-10-04T18:51:54.8994646Z   token: ***
2021-10-04T18:51:54.8994955Z ##[endgroup]
2021-10-04T18:51:55.0555469Z Setup go stable version spec 1.16.x
2021-10-04T18:51:55.0596667Z Found in cache @ /opt/hostedtoolcache/go/1.16.8/x64
2021-10-04T18:51:55.0604734Z Added go to the path
2021-10-04T18:51:55.7809798Z Successfully setup go version 1.16.x
2021-10-04T18:51:55.7947644Z go version go1.16.8 linux/amd64
2021-10-04T18:51:55.7948127Z 
2021-10-04T18:51:55.7949097Z ##[group]go env
2021-10-04T18:51:55.9124764Z GO111MODULE=""
2021-10-04T18:51:55.9125440Z GOARCH="amd64"
2021-10-04T18:51:55.9125858Z GOBIN=""
2021-10-04T18:51:55.9126883Z GOCACHE="/home/runner/.cache/go-build"
2021-10-04T18:51:55.9127473Z GOENV="/home/runner/.config/go/env"
2021-10-04T18:51:55.9127943Z GOEXE=""
2021-10-04T18:51:55.9128331Z GOFLAGS=""
2021-10-04T18:51:55.9128863Z GOHOSTARCH="amd64"
2021-10-04T18:51:55.9129329Z GOHOSTOS="linux"
2021-10-04T18:51:55.9129774Z GOINSECURE=""
2021-10-04T18:51:55.9130270Z GOMODCACHE="/home/runner/go/pkg/mod"
2021-10-04T18:51:55.9130773Z GONOPROXY=""
2021-10-04T18:51:55.9131191Z GONOSUMDB=""
2021-10-04T18:51:55.9131607Z GOOS="linux"
2021-10-04T18:51:55.9132053Z GOPATH="/home/runner/go"
2021-10-04T18:51:55.9132497Z GOPRIVATE=""
2021-10-04T18:51:55.9133175Z GOPROXY="https://proxy.golang.org,direct"
2021-10-04T18:51:55.9133936Z GOROOT="/opt/hostedtoolcache/go/1.16.8/x64"
2021-10-04T18:51:55.9134485Z GOSUMDB="sum.golang.org"
2021-10-04T18:51:55.9134986Z GOTMPDIR=""
2021-10-04T18:51:55.9135559Z GOTOOLDIR="/opt/hostedtoolcache/go/1.16.8/x64/pkg/tool/linux_amd64"
2021-10-04T18:51:55.9136441Z GOVCS=""
2021-10-04T18:51:55.9137090Z GOVERSION="go1.16.8"
2021-10-04T18:51:55.9137524Z GCCGO="gccgo"
2021-10-04T18:51:55.9137921Z AR="ar"
2021-10-04T18:51:55.9138294Z CC="gcc"
2021-10-04T18:51:55.9139333Z CXX="g++"
2021-10-04T18:51:55.9139766Z CGO_ENABLED="1"
2021-10-04T18:51:55.9140194Z GOMOD="/dev/null"
2021-10-04T18:51:55.9140974Z CGO_CFLAGS="-g -O2"
2021-10-04T18:51:55.9141422Z CGO_CPPFLAGS=""
2021-10-04T18:51:55.9141992Z CGO_CXXFLAGS="-g -O2"
2021-10-04T18:51:55.9142566Z CGO_FFLAGS="-g -O2"
2021-10-04T18:51:55.9143111Z CGO_LDFLAGS="-g -O2"
2021-10-04T18:51:55.9143718Z PKG_CONFIG="pkg-config"
2021-10-04T18:51:55.9144913Z GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build2954492157=/tmp/go-build -gno-record-gcc-switches"
2021-10-04T18:51:55.9146847Z 
2021-10-04T18:51:55.9148252Z ##[endgroup]
2021-10-04T18:51:55.9267515Z ##[group]Run actions/checkout@v2
2021-10-04T18:51:55.9267871Z with:
2021-10-04T18:51:55.9268189Z   fetch-depth: 0
2021-10-04T18:51:55.9268565Z   repository: knative/serving
2021-10-04T18:51:55.9269327Z   token: ***
2021-10-04T18:51:55.9269651Z   ssh-strict: true
2021-10-04T18:51:55.9270060Z   persist-credentials: true
2021-10-04T18:51:55.9270459Z   clean: true
2021-10-04T18:51:55.9270742Z   lfs: false
2021-10-04T18:51:55.9271061Z   submodules: false
2021-10-04T18:51:55.9271360Z env:
2021-10-04T18:51:55.9271745Z   GOROOT: /opt/hostedtoolcache/go/1.16.8/x64
2021-10-04T18:51:55.9272154Z ##[endgroup]
2021-10-04T18:51:56.7409878Z Syncing repository: knative/serving
2021-10-04T18:51:56.7410881Z ##[group]Getting Git version info
2021-10-04T18:51:56.7411763Z Working directory is '/home/runner/work/serving/serving'
2021-10-04T18:51:56.7412695Z [command]/usr/bin/git version
2021-10-04T18:51:56.7413069Z git version 2.33.0
2021-10-04T18:51:56.7414454Z ##[endgroup]
2021-10-04T18:51:56.7415086Z Deleting the contents of '/home/runner/work/serving/serving'
2021-10-04T18:51:56.7416991Z ##[group]Initializing the repository
2021-10-04T18:51:56.7417539Z [command]/usr/bin/git init /home/runner/work/serving/serving
2021-10-04T18:51:56.7418321Z hint: Using 'master' as the name for the initial branch. This default branch name
2021-10-04T18:51:56.7418984Z hint: is subject to change. To configure the initial branch name to use in all
2021-10-04T18:51:56.7419644Z hint: of your new repositories, which will suppress this warning, call:
2021-10-04T18:51:56.7420105Z hint: 
2021-10-04T18:51:56.7420680Z hint: 	git config --global init.defaultBranch <name>
2021-10-04T18:51:56.7421110Z hint: 
2021-10-04T18:51:56.7421689Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2021-10-04T18:51:56.7422475Z hint: 'development'. The just-created branch can be renamed via this command:
2021-10-04T18:51:56.7422974Z hint: 
2021-10-04T18:51:56.7423406Z hint: 	git branch -m <name>
2021-10-04T18:51:56.7423942Z Initialized empty Git repository in /home/runner/work/serving/serving/.git/
2021-10-04T18:51:56.7424682Z [command]/usr/bin/git remote add origin https://github.com/knative/serving
2021-10-04T18:51:56.7425621Z ##[endgroup]
2021-10-04T18:51:56.7426469Z ##[group]Disabling automatic garbage collection
2021-10-04T18:51:56.7427096Z [command]/usr/bin/git config --local gc.auto 0
2021-10-04T18:51:56.7427892Z ##[endgroup]
2021-10-04T18:51:56.7429919Z ##[group]Setting up auth
2021-10-04T18:51:56.7430637Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2021-10-04T18:51:56.7431919Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2021-10-04T18:51:56.7433130Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2021-10-04T18:51:56.7434637Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2021-10-04T18:51:56.7436119Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2021-10-04T18:51:56.7437338Z ##[endgroup]
2021-10-04T18:51:56.7438144Z ##[group]Fetching the repository
2021-10-04T18:51:56.7439515Z [command]/usr/bin/git -c protocol.version=2 fetch --prune --progress --no-recurse-submodules origin +refs/heads/*:refs/remotes/origin/* +refs/tags/*:refs/tags/* +03820b8aa7dbf633730ad88d9968b460195aa600:refs/remotes/pull/12036/merge
2021-10-04T18:51:56.7440628Z remote: Enumerating objects: 121410, done.        
2021-10-04T18:51:56.7441101Z remote: Counting objects:   0% (1/2325)        
2021-10-04T18:51:56.7441525Z remote: Counting objects:   1% (24/2325)        
2021-10-04T18:51:56.7441957Z remote: Counting objects:   2% (47/2325)        
2021-10-04T18:51:56.7442378Z remote: Counting objects:   3% (70/2325)        
2021-10-04T18:51:56.7442930Z remote: Counting objects:   4% (93/2325)        
2021-10-04T18:51:56.7443375Z remote: Counting objects:   5% (117/2325)        
2021-10-04T18:51:56.7443796Z remote: Counting objects:   6% (140/2325)        
2021-10-04T18:51:56.7444240Z remote: Counting objects:   7% (163/2325)        
2021-10-04T18:51:56.7444667Z remote: Counting objects:   8% (186/2325)        
2021-10-04T18:51:56.7445104Z remote: Counting objects:   9% (210/2325)        
2021-10-04T18:51:56.7445541Z remote: Counting objects:  10% (233/2325)        
2021-10-04T18:51:56.7446001Z remote: Counting objects:  11% (256/2325)        
2021-10-04T18:51:56.7446434Z remote: Counting objects:  12% (279/2325)        
2021-10-04T18:51:56.7446857Z remote: Counting objects:  13% (303/2325)        
2021-10-04T18:51:56.7447296Z remote: Counting objects:  14% (326/2325)        
2021-10-04T18:51:56.7447724Z remote: Counting objects:  15% (349/2325)        
2021-10-04T18:51:56.7448147Z remote: Counting objects:  16% (372/2325)        
2021-10-04T18:51:56.7448574Z remote: Counting objects:  17% (396/2325)        
2021-10-04T18:51:56.7448991Z remote: Counting objects:  18% (419/2325)        
2021-10-04T18:51:56.7449424Z remote: Counting objects:  19% (442/2325)        
2021-10-04T18:51:56.7449898Z remote: Counting objects:  20% (465/2325)        
2021-10-04T18:51:56.7450317Z remote: Counting objects:  21% (489/2325)        
2021-10-04T18:51:56.7450750Z remote: Counting objects:  22% (512/2325)        
2021-10-04T18:51:56.7451172Z remote: Counting objects:  23% (535/2325)        
2021-10-04T18:51:56.7451604Z remote: Counting objects:  24% (558/2325)        
2021-10-04T18:51:56.7452070Z remote: Counting objects:  25% (582/2325)        
2021-10-04T18:51:56.7452506Z remote: Counting objects:  26% (605/2325)        
2021-10-04T18:51:56.7452944Z remote: Counting objects:  27% (628/2325)        
2021-10-04T18:51:56.7453364Z remote: Counting objects:  28% (651/2325)        
2021-10-04T18:51:56.7453801Z remote: Counting objects:  29% (675/2325)        
2021-10-04T18:51:56.7454406Z remote: Counting objects:  30% (698/2325)        
2021-10-04T18:51:56.7454826Z remote: Counting objects:  31% (721/2325)        
2021-10-04T18:51:56.7455258Z remote: Counting objects:  32% (744/2325)        
2021-10-04T18:51:56.7455800Z remote: Counting objects:  33% (768/2325)        
2021-10-04T18:51:56.7456238Z remote: Counting objects:  34% (791/2325)        
2021-10-04T18:51:56.7456732Z remote: Counting objects:  35% (814/2325)        
2021-10-04T18:51:56.7457189Z remote: Counting objects:  36% (837/2325)        
2021-10-04T18:51:56.7457698Z remote: Counting objects:  37% (861/2325)        
2021-10-04T18:51:56.7458195Z remote: Counting objects:  38% (884/2325)        
2021-10-04T18:51:56.7458716Z remote: Counting objects:  39% (907/2325)        
2021-10-04T18:51:56.7459163Z remote: Counting objects:  40% (930/2325)        
2021-10-04T18:51:56.7459660Z remote: Counting objects:  41% (954/2325)        
2021-10-04T18:51:56.7460137Z remote: Counting objects:  42% (977/2325)        
2021-10-04T18:51:56.7460638Z remote: Counting objects:  43% (1000/2325)        
2021-10-04T18:51:56.7461158Z remote: Counting objects:  44% (1023/2325)        
2021-10-04T18:51:56.7461617Z remote: Counting objects:  45% (1047/2325)        
2021-10-04T18:51:56.7462232Z remote: Counting objects:  46% (1070/2325)        
2021-10-04T18:51:56.7462748Z remote: Counting objects:  47% (1093/2325)        
2021-10-04T18:51:56.7463262Z remote: Counting objects:  48% (1116/2325)        
2021-10-04T18:51:56.7463719Z remote: Counting objects:  49% (1140/2325)        
2021-10-04T18:51:56.7464214Z remote: Counting objects:  50% (1163/2325)        
2021-10-04T18:51:56.7464710Z remote: Counting objects:  51% (1186/2325)        
2021-10-04T18:51:56.7465180Z remote: Counting objects:  52% (1209/2325)        
2021-10-04T18:51:56.7465691Z remote: Counting objects:  53% (1233/2325)        
2021-10-04T18:51:56.7466146Z remote: Counting objects:  54% (1256/2325)        
2021-10-04T18:51:56.7466755Z remote: Counting objects:  55% (1279/2325)        
2021-10-04T18:51:56.7467266Z remote: Counting objects:  56% (1302/2325)        
2021-10-04T18:51:56.7467738Z remote: Counting objects:  57% (1326/2325)        
2021-10-04T18:51:56.7468255Z remote: Counting objects:  58% (1349/2325)        
2021-10-04T18:51:56.7468709Z remote: Counting objects:  59% (1372/2325)        
2021-10-04T18:51:56.7469209Z remote: Counting objects:  60% (1395/2325)        
2021-10-04T18:51:56.7469706Z remote: Counting objects:  61% (1419/2325)        
2021-10-04T18:51:56.7470183Z remote: Counting objects:  62% (1442/2325)        
2021-10-04T18:51:56.7470681Z remote: Counting objects:  63% (1465/2325)        
2021-10-04T18:51:56.7471223Z remote: Counting objects:  64% (1488/2325)        
2021-10-04T18:51:56.7471725Z remote: Counting objects:  65% (1512/2325)        
2021-10-04T18:51:56.7472237Z remote: Counting objects:  66% (1535/2325)        
2021-10-04T18:51:56.7472720Z remote: Counting objects:  67% (1558/2325)        
2021-10-04T18:51:56.7473221Z remote: Counting objects:  68% (1581/2325)        
2021-10-04T18:51:56.7473672Z remote: Counting objects:  69% (1605/2325)        
2021-10-04T18:51:56.7474165Z remote: Counting objects:  70% (1628/2325)        
2021-10-04T18:51:56.7474624Z remote: Counting objects:  71% (1651/2325)        
2021-10-04T18:51:56.7475142Z remote: Counting objects:  72% (1674/2325)        
2021-10-04T18:51:56.7475641Z remote: Counting objects:  73% (1698/2325)        
2021-10-04T18:51:56.7476090Z remote: Counting objects:  74% (1721/2325)        
2021-10-04T18:51:56.7476584Z remote: Counting objects:  75% (1744/2325)        
2021-10-04T18:51:56.7477037Z remote: Counting objects:  76% (1767/2325)        
2021-10-04T18:51:56.7477565Z remote: Counting objects:  77% (1791/2325)        
2021-10-04T18:51:56.7478067Z remote: Counting objects:  78% (1814/2325)        
2021-10-04T18:51:56.7478518Z remote: Counting objects:  79% (1837/2325)        
2021-10-04T18:51:56.7479010Z remote: Counting objects:  80% (1860/2325)        
2021-10-04T18:51:56.7479479Z remote: Counting objects:  81% (1884/2325)        
2021-10-04T18:51:56.7479985Z remote: Counting objects:  82% (1907/2325)        
2021-10-04T18:51:56.7480482Z remote: Counting objects:  83% (1930/2325)        
2021-10-04T18:51:56.7480934Z remote: Counting objects:  84% (1953/2325)        
2021-10-04T18:51:56.7481426Z remote: Counting objects:  85% (1977/2325)        
2021-10-04T18:51:56.7481912Z remote: Counting objects:  86% (2000/2325)        
2021-10-04T18:51:56.7482422Z remote: Counting objects:  87% (2023/2325)        
2021-10-04T18:51:56.7482918Z remote: Counting objects:  88% (2046/2325)        
2021-10-04T18:51:56.7483370Z remote: Counting objects:  89% (2070/2325)        
2021-10-04T18:51:56.7483877Z remote: Counting objects:  90% (2093/2325)        
2021-10-04T18:51:56.7484330Z remote: Counting objects:  91% (2116/2325)        
2021-10-04T18:51:56.7484839Z remote: Counting objects:  92% (2139/2325)        
2021-10-04T18:51:56.7485296Z remote: Counting objects:  93% (2163/2325)        
2021-10-04T18:51:56.7485795Z remote: Counting objects:  94% (2186/2325)        
2021-10-04T18:51:56.7486303Z remote: Counting objects:  95% (2209/2325)        
2021-10-04T18:51:56.7486754Z remote: Counting objects:  96% (2232/2325)        
2021-10-04T18:51:56.7487361Z remote: Counting objects:  97% (2256/2325)        
2021-10-04T18:51:56.7487819Z remote: Counting objects:  98% (2279/2325)        
2021-10-04T18:51:56.7488321Z remote: Counting objects:  99% (2302/2325)        
2021-10-04T18:51:56.7488888Z remote: Counting objects: 100% (2325/2325)        
2021-10-04T18:51:56.7489360Z remote: Counting objects: 100% (2325/2325), done.        
2021-10-04T18:51:56.7489911Z remote: Compressing objects:   0% (1/1123)        
2021-10-04T18:51:56.7490404Z remote: Compressing objects:   1% (12/1123)        
2021-10-04T18:51:56.7490951Z remote: Compressing objects:   2% (23/1123)        
2021-10-04T18:51:56.7491541Z remote: Compressing objects:   3% (34/1123)        
2021-10-04T18:51:56.7492086Z remote: Compressing objects:   4% (45/1123)        
2021-10-04T18:51:56.7492650Z remote: Compressing objects:   5% (57/1123)        
2021-10-04T18:51:56.7493145Z remote: Compressing objects:   6% (68/1123)        
2021-10-04T18:51:56.7493694Z remote: Compressing objects:   7% (79/1123)        
2021-10-04T18:51:56.7494228Z remote: Compressing objects:   8% (90/1123)        
2021-10-04T18:51:56.7494723Z remote: Compressing objects:   9% (102/1123)        
2021-10-04T18:51:56.7495276Z remote: Compressing objects:  10% (113/1123)        
2021-10-04T18:51:56.7495968Z remote: Compressing objects:  11% (124/1123)        
2021-10-04T18:51:56.7496527Z remote: Compressing objects:  12% (135/1123)        
2021-10-04T18:51:56.7497024Z remote: Compressing objects:  13% (146/1123)        
2021-10-04T18:51:56.7497558Z remote: Compressing objects:  14% (158/1123)        
2021-10-04T18:51:56.7498119Z remote: Compressing objects:  15% (169/1123)        
2021-10-04T18:51:56.7498627Z remote: Compressing objects:  16% (180/1123)        
2021-10-04T18:51:56.7499170Z remote: Compressing objects:  17% (191/1123)        
2021-10-04T18:51:56.7499666Z remote: Compressing objects:  18% (203/1123)        
2021-10-04T18:51:56.7500203Z remote: Compressing objects:  19% (214/1123)        
2021-10-04T18:51:56.7500760Z remote: Compressing objects:  20% (225/1123)        
2021-10-04T18:51:56.7501264Z remote: Compressing objects:  21% (236/1123)        
2021-10-04T18:51:56.7501801Z remote: Compressing objects:  22% (248/1123)        
2021-10-04T18:51:56.7502314Z remote: Compressing objects:  23% (259/1123)        
2021-10-04T18:51:56.7502846Z remote: Compressing objects:  24% (270/1123)        
2021-10-04T18:51:56.7503411Z remote: Compressing objects:  25% (281/1123)        
2021-10-04T18:51:56.7503903Z remote: Compressing objects:  26% (292/1123)        
2021-10-04T18:51:56.7504490Z remote: Compressing objects:  27% (304/1123)        
2021-10-04T18:51:56.7504980Z remote: Compressing objects:  28% (315/1123)        
2021-10-04T18:51:56.7505518Z remote: Compressing objects:  29% (326/1123)        
2021-10-04T18:51:56.7506074Z remote: Compressing objects:  30% (337/1123)        
2021-10-04T18:51:56.7506570Z remote: Compressing objects:  31% (349/1123)        
2021-10-04T18:51:56.7507104Z remote: Compressing objects:  32% (360/1123)        
2021-10-04T18:51:56.7507597Z remote: Compressing objects:  33% (371/1123)        
2021-10-04T18:51:56.7508132Z remote: Compressing objects:  34% (382/1123)        
2021-10-04T18:51:56.7508896Z remote: Compressing objects:  35% (394/1123)        
2021-10-04T18:51:56.7509456Z remote: Compressing objects:  36% (405/1123)        
2021-10-04T18:51:56.7509992Z remote: Compressing objects:  37% (416/1123)        
2021-10-04T18:51:56.7510481Z remote: Compressing objects:  38% (427/1123)        
2021-10-04T18:51:56.7511017Z remote: Compressing objects:  39% (438/1123)        
2021-10-04T18:51:56.7511565Z remote: Compressing objects:  40% (450/1123)        
2021-10-04T18:51:56.7512118Z remote: Compressing objects:  41% (461/1123)        
2021-10-04T18:51:56.7512659Z remote: Compressing objects:  42% (472/1123)        
2021-10-04T18:51:56.7513167Z remote: Compressing objects:  43% (483/1123)        
2021-10-04T18:51:56.7513713Z remote: Compressing objects:  44% (495/1123)        
2021-10-04T18:51:56.7514315Z remote: Compressing objects:  45% (506/1123)        
2021-10-04T18:51:56.7514870Z remote: Compressing objects:  46% (517/1123)        
2021-10-04T18:51:56.7515408Z remote: Compressing objects:  47% (528/1123)        
2021-10-04T18:51:56.7515902Z remote: Compressing objects:  48% (540/1123)        
2021-10-04T18:51:56.7516457Z remote: Compressing objects:  49% (551/1123)        
2021-10-04T18:51:56.7516948Z remote: Compressing objects:  50% (562/1123)        
2021-10-04T18:51:56.7517500Z remote: Compressing objects:  51% (573/1123)        
2021-10-04T18:51:56.7517992Z remote: Compressing objects:  52% (584/1123)        
2021-10-04T18:51:56.7518528Z remote: Compressing objects:  53% (596/1123)        
2021-10-04T18:51:56.7519141Z remote: Compressing objects:  54% (607/1123)        
2021-10-04T18:51:56.7519640Z remote: Compressing objects:  55% (618/1123)        
2021-10-04T18:51:56.7520206Z remote: Compressing objects:  56% (629/1123)        
2021-10-04T18:51:56.7520744Z remote: Compressing objects:  57% (641/1123)        
2021-10-04T18:51:56.7521295Z remote: Compressing objects:  58% (652/1123)        
2021-10-04T18:51:56.7521786Z remote: Compressing objects:  59% (663/1123)        
2021-10-04T18:51:56.7522338Z remote: Compressing objects:  60% (674/1123)        
2021-10-04T18:51:56.7522836Z remote: Compressing objects:  61% (686/1123)        
2021-10-04T18:51:56.7523398Z remote: Compressing objects:  62% (697/1123)        
2021-10-04T18:51:56.7523929Z remote: Compressing objects:  63% (708/1123)        
2021-10-04T18:51:56.7524421Z remote: Compressing objects:  64% (719/1123)        
2021-10-04T18:51:56.7524973Z remote: Compressing objects:  65% (730/1123)        
2021-10-04T18:51:56.7525510Z remote: Compressing objects:  66% (742/1123)        
2021-10-04T18:51:56.7526060Z remote: Compressing objects:  67% (753/1123)        
2021-10-04T18:51:56.7526553Z remote: Compressing objects:  68% (764/1123)        
2021-10-04T18:51:56.7527207Z remote: Compressing objects:  69% (775/1123)        
2021-10-04T18:51:56.7527795Z remote: Compressing objects:  70% (787/1123)        
2021-10-04T18:51:56.7528287Z remote: Compressing objects:  71% (798/1123)        
2021-10-04T18:51:56.7528835Z remote: Compressing objects:  72% (809/1123)        
2021-10-04T18:51:56.7529326Z remote: Compressing objects:  73% (820/1123)        
2021-10-04T18:51:56.7529865Z remote: Compressing objects:  74% (832/1123)        
2021-10-04T18:51:56.7530412Z remote: Compressing objects:  75% (843/1123)        
2021-10-04T18:51:56.7530936Z remote: Compressing objects:  76% (854/1123)        
2021-10-04T18:51:56.7531488Z remote: Compressing objects:  77% (865/1123)        
2021-10-04T18:51:56.7531977Z remote: Compressing objects:  78% (876/1123)        
2021-10-04T18:51:56.7532516Z remote: Compressing objects:  79% (888/1123)        
2021-10-04T18:51:56.7533066Z remote: Compressing objects:  80% (899/1123)        
2021-10-04T18:51:56.7533570Z remote: Compressing objects:  81% (910/1123)        
2021-10-04T18:51:56.7534122Z remote: Compressing objects:  82% (921/1123)        
2021-10-04T18:51:56.7534615Z remote: Compressing objects:  83% (933/1123)        
2021-10-04T18:51:56.7535153Z remote: Compressing objects:  84% (944/1123)        
2021-10-04T18:51:56.7535804Z remote: Compressing objects:  85% (955/1123)        
2021-10-04T18:51:56.7536317Z remote: Compressing objects:  86% (966/1123)        
2021-10-04T18:51:56.7536850Z remote: Compressing objects:  87% (978/1123)        
2021-10-04T18:51:56.7537343Z remote: Compressing objects:  88% (989/1123)        
2021-10-04T18:51:56.7537878Z remote: Compressing objects:  89% (1000/1123)        
2021-10-04T18:51:56.7538374Z remote: Compressing objects:  90% (1011/1123)        
2021-10-04T18:51:56.7538941Z remote: Compressing objects:  91% (1022/1123)        
2021-10-04T18:51:56.7539484Z remote: Compressing objects:  92% (1034/1123)        
2021-10-04T18:51:56.7539977Z remote: Compressing objects:  93% (1045/1123)        
2021-10-04T18:51:56.7540521Z remote: Compressing objects:  94% (1056/1123)        
2021-10-04T18:51:56.7541119Z remote: Compressing objects:  95% (1067/1123)        
2021-10-04T18:51:56.7541675Z remote: Compressing objects:  96% (1079/1123)        
2021-10-04T18:51:56.7542210Z remote: Compressing objects:  97% (1090/1123)        
2021-10-04T18:51:56.7542702Z remote: Compressing objects:  98% (1101/1123)        
2021-10-04T18:51:56.7543240Z remote: Compressing objects:  99% (1112/1123)        
2021-10-04T18:51:56.7543746Z remote: Compressing objects: 100% (1123/1123)        
2021-10-04T18:51:56.7544348Z remote: Compressing objects: 100% (1123/1123), done.        
2021-10-04T18:51:56.7544863Z Receiving objects:   0% (1/121410)
2021-10-04T18:51:56.7591640Z Receiving objects:   1% (1215/121410)
2021-10-04T18:51:56.7766983Z Receiving objects:   2% (2429/121410)
2021-10-04T18:51:56.7893157Z Receiving objects:   3% (3643/121410)
2021-10-04T18:51:56.8017704Z Receiving objects:   4% (4857/121410)
2021-10-04T18:51:56.8205846Z Receiving objects:   5% (6071/121410)
2021-10-04T18:51:56.8449539Z Receiving objects:   6% (7285/121410)
2021-10-04T18:51:56.8677080Z Receiving objects:   7% (8499/121410)
2021-10-04T18:51:56.8981327Z Receiving objects:   8% (9713/121410)
2021-10-04T18:51:56.9377659Z Receiving objects:   9% (10927/121410)
2021-10-04T18:51:56.9790584Z Receiving objects:  10% (12141/121410)
2021-10-04T18:51:57.0193052Z Receiving objects:  11% (13356/121410)
2021-10-04T18:51:57.0930145Z Receiving objects:  12% (14570/121410)
2021-10-04T18:51:57.1412358Z Receiving objects:  13% (15784/121410)
2021-10-04T18:51:57.1538102Z Receiving objects:  14% (16998/121410)
2021-10-04T18:51:57.1711352Z Receiving objects:  15% (18212/121410)
2021-10-04T18:51:57.1871787Z Receiving objects:  16% (19426/121410)
2021-10-04T18:51:57.2013054Z Receiving objects:  17% (20640/121410)
2021-10-04T18:51:57.2221195Z Receiving objects:  18% (21854/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.2359977Z Receiving objects:  19% (23068/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.2537582Z Receiving objects:  20% (24282/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.2766664Z Receiving objects:  21% (25497/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.2999503Z Receiving objects:  22% (26711/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.3172695Z Receiving objects:  23% (27925/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.3259220Z Receiving objects:  24% (29139/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.3404296Z Receiving objects:  25% (30353/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.3507765Z Receiving objects:  26% (31567/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.3970419Z Receiving objects:  27% (32781/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.5575639Z Receiving objects:  28% (33995/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.5769639Z Receiving objects:  29% (35209/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.5866724Z Receiving objects:  30% (36423/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.6057966Z Receiving objects:  31% (37638/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.6094626Z Receiving objects:  32% (38852/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.6527404Z Receiving objects:  33% (40066/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.6977414Z Receiving objects:  34% (41280/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.6999471Z Receiving objects:  34% (42028/121410), 15.52 MiB | 31.02 MiB/s
2021-10-04T18:51:57.7090589Z Receiving objects:  35% (42494/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.7245149Z Receiving objects:  36% (43708/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.7375906Z Receiving objects:  37% (44922/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.7488018Z Receiving objects:  38% (46136/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.7631877Z Receiving objects:  39% (47350/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.7878464Z Receiving objects:  40% (48564/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.7990624Z Receiving objects:  41% (49779/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8136692Z Receiving objects:  42% (50993/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8207261Z Receiving objects:  43% (52207/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8306940Z Receiving objects:  44% (53421/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8424463Z Receiving objects:  45% (54635/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8521636Z Receiving objects:  46% (55849/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8627183Z Receiving objects:  47% (57063/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8707116Z Receiving objects:  48% (58277/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8856670Z Receiving objects:  49% (59491/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.8971761Z Receiving objects:  50% (60705/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9051909Z Receiving objects:  51% (61920/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9188650Z Receiving objects:  52% (63134/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9271567Z Receiving objects:  53% (64348/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9339127Z Receiving objects:  54% (65562/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9386014Z Receiving objects:  55% (66776/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9464189Z Receiving objects:  56% (67990/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9520527Z Receiving objects:  57% (69204/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9795565Z Receiving objects:  58% (70418/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:57.9960136Z Receiving objects:  59% (71632/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0015162Z Receiving objects:  60% (72846/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0090898Z Receiving objects:  61% (74061/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0178380Z Receiving objects:  62% (75275/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0281912Z Receiving objects:  63% (76489/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0371984Z Receiving objects:  64% (77703/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0429706Z Receiving objects:  65% (78917/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0514687Z Receiving objects:  66% (80131/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0597447Z Receiving objects:  67% (81345/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0660187Z Receiving objects:  68% (82559/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0737038Z Receiving objects:  69% (83773/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0812489Z Receiving objects:  70% (84987/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0860023Z Receiving objects:  71% (86202/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.0921066Z Receiving objects:  72% (87416/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1046584Z Receiving objects:  73% (88630/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1124372Z Receiving objects:  74% (89844/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1193240Z Receiving objects:  75% (91058/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1261218Z Receiving objects:  76% (92272/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1321316Z Receiving objects:  77% (93486/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1391741Z Receiving objects:  78% (94700/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1501948Z Receiving objects:  79% (95914/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1604303Z Receiving objects:  80% (97128/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1676286Z Receiving objects:  81% (98343/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.1779870Z Receiving objects:  82% (99557/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.2631243Z Receiving objects:  83% (100771/121410), 35.04 MiB | 35.03 MiB/s
2021-10-04T18:51:58.2740821Z Receiving objects:  84% (101985/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.2847170Z Receiving objects:  85% (103199/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.3278934Z Receiving objects:  86% (104413/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.3338766Z Receiving objects:  87% (105627/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.3408310Z Receiving objects:  88% (106841/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.3474110Z Receiving objects:  89% (108055/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.3707929Z Receiving objects:  90% (109269/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.3797539Z Receiving objects:  91% (110484/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.3876873Z Receiving objects:  92% (111698/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.4211395Z Receiving objects:  93% (112912/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.4418872Z Receiving objects:  94% (114126/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.4483762Z Receiving objects:  95% (115340/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.4561422Z Receiving objects:  96% (116554/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.4969346Z Receiving objects:  97% (117768/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.5045356Z Receiving objects:  98% (118982/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.5644943Z Receiving objects:  99% (120196/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.5646670Z remote: Total 121410 (delta 1122), reused 2032 (delta 1043), pack-reused 119085        
2021-10-04T18:51:58.5664134Z Receiving objects: 100% (121410/121410), 57.57 MiB | 38.38 MiB/s
2021-10-04T18:51:58.5664988Z Receiving objects: 100% (121410/121410), 72.35 MiB | 38.71 MiB/s, done.
2021-10-04T18:51:58.5907401Z Resolving deltas:   0% (0/82167)
2021-10-04T18:51:58.6079797Z Resolving deltas:   1% (822/82167)
2021-10-04T18:51:58.6295499Z Resolving deltas:   2% (1644/82167)
2021-10-04T18:51:58.6471406Z Resolving deltas:   3% (2466/82167)
2021-10-04T18:51:58.6665037Z Resolving deltas:   4% (3287/82167)
2021-10-04T18:51:58.6893148Z Resolving deltas:   5% (4109/82167)
2021-10-04T18:51:58.7032291Z Resolving deltas:   6% (4931/82167)
2021-10-04T18:51:58.7138346Z Resolving deltas:   7% (5752/82167)
2021-10-04T18:51:58.7214166Z Resolving deltas:   8% (6574/82167)
2021-10-04T18:51:58.7430792Z Resolving deltas:   9% (7396/82167)
2021-10-04T18:51:58.7590361Z Resolving deltas:  10% (8217/82167)
2021-10-04T18:51:58.7947487Z Resolving deltas:  11% (9039/82167)
2021-10-04T18:51:58.8156508Z Resolving deltas:  12% (9861/82167)
2021-10-04T18:51:58.8269857Z Resolving deltas:  13% (10682/82167)
2021-10-04T18:51:58.8393195Z Resolving deltas:  14% (11504/82167)
2021-10-04T18:51:58.8469368Z Resolving deltas:  15% (12326/82167)
2021-10-04T18:51:58.8680498Z Resolving deltas:  16% (13147/82167)
2021-10-04T18:51:58.9459103Z Resolving deltas:  17% (13969/82167)
2021-10-04T18:51:58.9644239Z Resolving deltas:  18% (14791/82167)
2021-10-04T18:51:58.9834268Z Resolving deltas:  19% (15612/82167)
2021-10-04T18:51:59.0269019Z Resolving deltas:  20% (16434/82167)
2021-10-04T18:51:59.0771717Z Resolving deltas:  21% (17256/82167)
2021-10-04T18:51:59.1734137Z Resolving deltas:  22% (18077/82167)
2021-10-04T18:51:59.2020644Z Resolving deltas:  23% (18899/82167)
2021-10-04T18:51:59.2169325Z Resolving deltas:  24% (19721/82167)
2021-10-04T18:51:59.2266772Z Resolving deltas:  25% (20542/82167)
2021-10-04T18:51:59.2341476Z Resolving deltas:  26% (21364/82167)
2021-10-04T18:51:59.2437871Z Resolving deltas:  27% (22186/82167)
2021-10-04T18:51:59.2580193Z Resolving deltas:  28% (23007/82167)
2021-10-04T18:51:59.2698510Z Resolving deltas:  29% (23829/82167)
2021-10-04T18:51:59.2787494Z Resolving deltas:  30% (24651/82167)
2021-10-04T18:51:59.2901988Z Resolving deltas:  31% (25472/82167)
2021-10-04T18:51:59.3047710Z Resolving deltas:  32% (26294/82167)
2021-10-04T18:51:59.3338425Z Resolving deltas:  33% (27116/82167)
2021-10-04T18:51:59.3458703Z Resolving deltas:  34% (27937/82167)
2021-10-04T18:51:59.4313250Z Resolving deltas:  35% (28759/82167)
2021-10-04T18:51:59.4396442Z Resolving deltas:  36% (29581/82167)
2021-10-04T18:51:59.4505695Z Resolving deltas:  37% (30402/82167)
2021-10-04T18:51:59.4563380Z Resolving deltas:  38% (31224/82167)
2021-10-04T18:51:59.4723754Z Resolving deltas:  39% (32046/82167)
2021-10-04T18:51:59.4786104Z Resolving deltas:  40% (32867/82167)
2021-10-04T18:51:59.4983697Z Resolving deltas:  41% (33689/82167)
2021-10-04T18:51:59.5368655Z Resolving deltas:  42% (34511/82167)
2021-10-04T18:51:59.5517429Z Resolving deltas:  43% (35332/82167)
2021-10-04T18:51:59.5630253Z Resolving deltas:  44% (36154/82167)
2021-10-04T18:51:59.5713083Z Resolving deltas:  45% (36976/82167)
2021-10-04T18:51:59.5765807Z Resolving deltas:  46% (37797/82167)
2021-10-04T18:51:59.5981525Z Resolving deltas:  46% (37893/82167)
2021-10-04T18:51:59.6154786Z Resolving deltas:  47% (38619/82167)
2021-10-04T18:51:59.6272740Z Resolving deltas:  48% (39441/82167)
2021-10-04T18:51:59.6964620Z Resolving deltas:  49% (40262/82167)
2021-10-04T18:51:59.7203734Z Resolving deltas:  50% (41084/82167)
2021-10-04T18:51:59.7318277Z Resolving deltas:  51% (41906/82167)
2021-10-04T18:51:59.7415099Z Resolving deltas:  52% (42727/82167)
2021-10-04T18:51:59.7443619Z Resolving deltas:  53% (43549/82167)
2021-10-04T18:51:59.7472037Z Resolving deltas:  54% (44371/82167)
2021-10-04T18:51:59.7526503Z Resolving deltas:  55% (45192/82167)
2021-10-04T18:51:59.8002164Z Resolving deltas:  56% (46014/82167)
2021-10-04T18:51:59.8034777Z Resolving deltas:  57% (46836/82167)
2021-10-04T18:51:59.8152270Z Resolving deltas:  58% (47657/82167)
2021-10-04T18:51:59.8184482Z Resolving deltas:  59% (48480/82167)
2021-10-04T18:51:59.8307494Z Resolving deltas:  60% (49301/82167)
2021-10-04T18:51:59.8527618Z Resolving deltas:  61% (50122/82167)
2021-10-04T18:51:59.8673545Z Resolving deltas:  62% (50944/82167)
2021-10-04T18:51:59.8715467Z Resolving deltas:  63% (51766/82167)
2021-10-04T18:51:59.8739493Z Resolving deltas:  64% (52587/82167)
2021-10-04T18:51:59.8765957Z Resolving deltas:  65% (53409/82167)
2021-10-04T18:51:59.8798356Z Resolving deltas:  66% (54231/82167)
2021-10-04T18:51:59.8831290Z Resolving deltas:  67% (55053/82167)
2021-10-04T18:51:59.8859826Z Resolving deltas:  68% (55874/82167)
2021-10-04T18:51:59.8882622Z Resolving deltas:  69% (56696/82167)
2021-10-04T18:51:59.8907772Z Resolving deltas:  70% (57517/82167)
2021-10-04T18:51:59.8951515Z Resolving deltas:  71% (58339/82167)
2021-10-04T18:51:59.9106627Z Resolving deltas:  72% (59161/82167)
2021-10-04T18:51:59.9270736Z Resolving deltas:  73% (59982/82167)
2021-10-04T18:51:59.9464293Z Resolving deltas:  74% (60804/82167)
2021-10-04T18:51:59.9508084Z Resolving deltas:  75% (61626/82167)
2021-10-04T18:51:59.9532450Z Resolving deltas:  76% (62447/82167)
2021-10-04T18:51:59.9596024Z Resolving deltas:  77% (63269/82167)
2021-10-04T18:51:59.9685798Z Resolving deltas:  78% (64091/82167)
2021-10-04T18:51:59.9730075Z Resolving deltas:  79% (64912/82167)
2021-10-04T18:51:59.9757646Z Resolving deltas:  80% (65734/82167)
2021-10-04T18:51:59.9786770Z Resolving deltas:  81% (66556/82167)
2021-10-04T18:51:59.9812516Z Resolving deltas:  82% (67377/82167)
2021-10-04T18:51:59.9849110Z Resolving deltas:  83% (68199/82167)
2021-10-04T18:51:59.9931613Z Resolving deltas:  84% (69021/82167)
2021-10-04T18:51:59.9961842Z Resolving deltas:  85% (69842/82167)
2021-10-04T18:51:59.9987363Z Resolving deltas:  86% (70664/82167)
2021-10-04T18:52:00.0016759Z Resolving deltas:  87% (71486/82167)
2021-10-04T18:52:00.0053212Z Resolving deltas:  88% (72307/82167)
2021-10-04T18:52:00.0108976Z Resolving deltas:  89% (73129/82167)
2021-10-04T18:52:00.0260645Z Resolving deltas:  90% (73951/82167)
2021-10-04T18:52:00.0300127Z Resolving deltas:  91% (74772/82167)
2021-10-04T18:52:00.0399477Z Resolving deltas:  92% (75594/82167)
2021-10-04T18:52:00.0439164Z Resolving deltas:  93% (76416/82167)
2021-10-04T18:52:00.0473324Z Resolving deltas:  94% (77237/82167)
2021-10-04T18:52:00.0512549Z Resolving deltas:  95% (78059/82167)
2021-10-04T18:52:00.0604165Z Resolving deltas:  96% (78881/82167)
2021-10-04T18:52:00.0742247Z Resolving deltas:  97% (79702/82167)
2021-10-04T18:52:00.0819568Z Resolving deltas:  98% (80524/82167)
2021-10-04T18:52:00.1105577Z Resolving deltas:  99% (81346/82167)
2021-10-04T18:52:00.1109141Z Resolving deltas: 100% (82167/82167)
2021-10-04T18:52:00.1109723Z Resolving deltas: 100% (82167/82167), done.
2021-10-04T18:52:00.7019760Z From https://github.com/knative/serving
2021-10-04T18:52:00.7022915Z  * [new branch]          main         -> origin/main
2021-10-04T18:52:00.7023606Z  * [new branch]          release-0.1  -> origin/release-0.1
2021-10-04T18:52:00.7024290Z  * [new branch]          release-0.10 -> origin/release-0.10
2021-10-04T18:52:00.7024991Z  * [new branch]          release-0.11 -> origin/release-0.11
2021-10-04T18:52:00.7026538Z  * [new branch]          release-0.12 -> origin/release-0.12
2021-10-04T18:52:00.7027429Z  * [new branch]          release-0.13 -> origin/release-0.13
2021-10-04T18:52:00.7029089Z  * [new branch]          release-0.14 -> origin/release-0.14
2021-10-04T18:52:00.7029766Z  * [new branch]          release-0.15 -> origin/release-0.15
2021-10-04T18:52:00.7031257Z  * [new branch]          release-0.16 -> origin/release-0.16
2021-10-04T18:52:00.7031954Z  * [new branch]          release-0.17 -> origin/release-0.17
2021-10-04T18:52:00.7032632Z  * [new branch]          release-0.18 -> origin/release-0.18
2021-10-04T18:52:00.7034095Z  * [new branch]          release-0.19 -> origin/release-0.19
2021-10-04T18:52:00.7034762Z  * [new branch]          release-0.2  -> origin/release-0.2
2021-10-04T18:52:00.7035446Z  * [new branch]          release-0.20 -> origin/release-0.20
2021-10-04T18:52:00.7036821Z  * [new branch]          release-0.21 -> origin/release-0.21
2021-10-04T18:52:00.7037569Z  * [new branch]          release-0.22 -> origin/release-0.22
2021-10-04T18:52:00.7038274Z  * [new branch]          release-0.23 -> origin/release-0.23
2021-10-04T18:52:00.7039686Z  * [new branch]          release-0.24 -> origin/release-0.24
2021-10-04T18:52:00.7040385Z  * [new branch]          release-0.25 -> origin/release-0.25
2021-10-04T18:52:00.7041080Z  * [new branch]          release-0.26 -> origin/release-0.26
2021-10-04T18:52:00.7042459Z  * [new branch]          release-0.3  -> origin/release-0.3
2021-10-04T18:52:00.7043140Z  * [new branch]          release-0.4  -> origin/release-0.4
2021-10-04T18:52:00.7043800Z  * [new branch]          release-0.5  -> origin/release-0.5
2021-10-04T18:52:00.7045215Z  * [new branch]          release-0.6  -> origin/release-0.6
2021-10-04T18:52:00.7045892Z  * [new branch]          release-0.7  -> origin/release-0.7
2021-10-04T18:52:00.7047188Z  * [new branch]          release-0.8  -> origin/release-0.8
2021-10-04T18:52:00.7047929Z  * [new branch]          release-0.9  -> origin/release-0.9
2021-10-04T18:52:00.7048486Z  * [new tag]             v0.1.0       -> v0.1.0
2021-10-04T18:52:00.7049657Z  * [new tag]             v0.1.1       -> v0.1.1
2021-10-04T18:52:00.7050195Z  * [new tag]             v0.10.0      -> v0.10.0
2021-10-04T18:52:00.7050664Z  * [new tag]             v0.10.1      -> v0.10.1
2021-10-04T18:52:00.7051153Z  * [new tag]             v0.11.0      -> v0.11.0
2021-10-04T18:52:00.7052347Z  * [new tag]             v0.11.1      -> v0.11.1
2021-10-04T18:52:00.7052835Z  * [new tag]             v0.11.2      -> v0.11.2
2021-10-04T18:52:00.7053313Z  * [new tag]             v0.12.0      -> v0.12.0
2021-10-04T18:52:00.7053782Z  * [new tag]             v0.12.1      -> v0.12.1
2021-10-04T18:52:00.7054338Z  * [new tag]             v0.13.0      -> v0.13.0
2021-10-04T18:52:00.7054801Z  * [new tag]             v0.13.1      -> v0.13.1
2021-10-04T18:52:00.7055281Z  * [new tag]             v0.13.2      -> v0.13.2
2021-10-04T18:52:00.7057263Z  * [new tag]             v0.13.3      -> v0.13.3
2021-10-04T18:52:00.7057735Z  * [new tag]             v0.14.0      -> v0.14.0
2021-10-04T18:52:00.7058220Z  * [new tag]             v0.14.1      -> v0.14.1
2021-10-04T18:52:00.7058684Z  * [new tag]             v0.14.2      -> v0.14.2
2021-10-04T18:52:00.7059163Z  * [new tag]             v0.14.3      -> v0.14.3
2021-10-04T18:52:00.7059820Z  * [new tag]             v0.15.0      -> v0.15.0
2021-10-04T18:52:00.7060428Z  * [new tag]             v0.15.1      -> v0.15.1
2021-10-04T18:52:00.7060912Z  * [new tag]             v0.15.2      -> v0.15.2
2021-10-04T18:52:00.7061372Z  * [new tag]             v0.15.3      -> v0.15.3
2021-10-04T18:52:00.7062790Z  * [new tag]             v0.16.0      -> v0.16.0
2021-10-04T18:52:00.7063390Z  * [new tag]             v0.17.0      -> v0.17.0
2021-10-04T18:52:00.7063868Z  * [new tag]             v0.17.1      -> v0.17.1
2021-10-04T18:52:00.7066961Z  * [new tag]             v0.17.2      -> v0.17.2
2021-10-04T18:52:00.7067444Z  * [new tag]             v0.17.3      -> v0.17.3
2021-10-04T18:52:00.7070084Z  * [new tag]             v0.17.4      -> v0.17.4
2021-10-04T18:52:00.7097578Z  * [new tag]             v0.18.0      -> v0.18.0
2021-10-04T18:52:00.7098596Z  * [new tag]             v0.18.1      -> v0.18.1
2021-10-04T18:52:00.7099222Z  * [new tag]             v0.18.2      -> v0.18.2
2021-10-04T18:52:00.7099826Z  * [new tag]             v0.18.3      -> v0.18.3
2021-10-04T18:52:00.7100421Z  * [new tag]             v0.19.0      -> v0.19.0
2021-10-04T18:52:00.7101014Z  * [new tag]             v0.2.0       -> v0.2.0
2021-10-04T18:52:00.7108116Z  * [new tag]             v0.2.1       -> v0.2.1
2021-10-04T18:52:00.7108604Z  * [new tag]             v0.2.2       -> v0.2.2
2021-10-04T18:52:00.7109080Z  * [new tag]             v0.2.3       -> v0.2.3
2021-10-04T18:52:00.7110980Z  * [new tag]             v0.20.0      -> v0.20.0
2021-10-04T18:52:00.7112000Z  * [new tag]             v0.20.1      -> v0.20.1
2021-10-04T18:52:00.7114232Z  * [new tag]             v0.20.2      -> v0.20.2
2021-10-04T18:52:00.7115656Z  * [new tag]             v0.21.0      -> v0.21.0
2021-10-04T18:52:00.7116236Z  * [new tag]             v0.21.1      -> v0.21.1
2021-10-04T18:52:00.7116814Z  * [new tag]             v0.22.0      -> v0.22.0
2021-10-04T18:52:00.7117316Z  * [new tag]             v0.22.1      -> v0.22.1
2021-10-04T18:52:00.7119824Z  * [new tag]             v0.22.2      -> v0.22.2
2021-10-04T18:52:00.7138396Z  * [new tag]             v0.22.3      -> v0.22.3
2021-10-04T18:52:00.7139042Z  * [new tag]             v0.23.0      -> v0.23.0
2021-10-04T18:52:00.7139575Z  * [new tag]             v0.23.1      -> v0.23.1
2021-10-04T18:52:00.7140116Z  * [new tag]             v0.23.2      -> v0.23.2
2021-10-04T18:52:00.7140641Z  * [new tag]             v0.24.0      -> v0.24.0
2021-10-04T18:52:00.7141179Z  * [new tag]             v0.24.1      -> v0.24.1
2021-10-04T18:52:00.7141714Z  * [new tag]             v0.25.0      -> v0.25.0
2021-10-04T18:52:00.7142232Z  * [new tag]             v0.25.1      -> v0.25.1
2021-10-04T18:52:00.7142761Z  * [new tag]             v0.26.0      -> v0.26.0
2021-10-04T18:52:00.7143295Z  * [new tag]             v0.3.0       -> v0.3.0
2021-10-04T18:52:00.7143830Z  * [new tag]             v0.4.0       -> v0.4.0
2021-10-04T18:52:00.7144350Z  * [new tag]             v0.4.1       -> v0.4.1
2021-10-04T18:52:00.7144884Z  * [new tag]             v0.5.0       -> v0.5.0
2021-10-04T18:52:00.7145421Z  * [new tag]             v0.5.1       -> v0.5.1
2021-10-04T18:52:00.7145939Z  * [new tag]             v0.5.2       -> v0.5.2
2021-10-04T18:52:00.7146468Z  * [new tag]             v0.6.0       -> v0.6.0
2021-10-04T18:52:00.7146979Z  * [new tag]             v0.6.1       -> v0.6.1
2021-10-04T18:52:00.7147503Z  * [new tag]             v0.7.0       -> v0.7.0
2021-10-04T18:52:00.7148035Z  * [new tag]             v0.7.1       -> v0.7.1
2021-10-04T18:52:00.7151607Z  * [new tag]             v0.8.0       -> v0.8.0
2021-10-04T18:52:00.7154252Z  * [new tag]             v0.8.1       -> v0.8.1
2021-10-04T18:52:00.7157413Z  * [new tag]             v0.9.0       -> v0.9.0
2021-10-04T18:52:00.7161993Z  * [new ref]             03820b8aa7dbf633730ad88d9968b460195aa600 -> pull/12036/merge
2021-10-04T18:52:00.7206828Z ##[endgroup]
2021-10-04T18:52:00.7207850Z ##[group]Determining the checkout info
2021-10-04T18:52:00.7210548Z ##[endgroup]
2021-10-04T18:52:00.7211442Z ##[group]Checking out the ref
2021-10-04T18:52:00.7212546Z [command]/usr/bin/git checkout --progress --force refs/remotes/pull/12036/merge
2021-10-04T18:52:01.2309349Z Note: switching to 'refs/remotes/pull/12036/merge'.
2021-10-04T18:52:01.2309795Z 
2021-10-04T18:52:01.2310604Z You are in 'detached HEAD' state. You can look around, make experimental
2021-10-04T18:52:01.2311456Z changes and commit them, and you can discard any commits you make in this
2021-10-04T18:52:01.2312250Z state without impacting any branches by switching back to a branch.
2021-10-04T18:52:01.2312727Z 
2021-10-04T18:52:01.2313288Z If you want to create a new branch to retain commits you create, you may
2021-10-04T18:52:01.2314222Z do so (now or later) by using -c with the switch command. Example:
2021-10-04T18:52:01.2314659Z 
2021-10-04T18:52:01.2315500Z   git switch -c <new-branch-name>
2021-10-04T18:52:01.2315867Z 
2021-10-04T18:52:01.2316285Z Or undo this operation with:
2021-10-04T18:52:01.2316614Z 
2021-10-04T18:52:01.2317094Z   git switch -
2021-10-04T18:52:01.2317379Z 
2021-10-04T18:52:01.2318034Z Turn off this advice by setting config variable advice.detachedHead to false
2021-10-04T18:52:01.2318590Z 
2021-10-04T18:52:01.2319578Z HEAD is now at 03820b8aa Merge 8c8fde1ccb8c5be89c59e358b530006e3edc2af3 into 643e3c4f03ffb3cc2aab5c910a370f113e851adc
2021-10-04T18:52:01.2337430Z ##[endgroup]
2021-10-04T18:52:01.2383832Z [command]/usr/bin/git log -1 --format='%H'
2021-10-04T18:52:01.2409165Z '03820b8aa7dbf633730ad88d9968b460195aa600'
2021-10-04T18:52:01.2561071Z ##[group]Run git remote add upstream https://github.com/knative/serving.git
2021-10-04T18:52:01.2562025Z [36;1mgit remote add upstream https://github.com/knative/serving.git[0m
2021-10-04T18:52:01.2562663Z [36;1mgit fetch upstream main[0m
2021-10-04T18:52:01.2563183Z [36;1mgit pull --no-rebase --no-commit upstream main[0m
2021-10-04T18:52:01.2601857Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2021-10-04T18:52:01.2602318Z env:
2021-10-04T18:52:01.2602754Z   GOROOT: /opt/hostedtoolcache/go/1.16.8/x64
2021-10-04T18:52:01.2603208Z ##[endgroup]
2021-10-04T18:52:01.9387210Z From https://github.com/knative/serving
2021-10-04T18:52:01.9388957Z  * branch                main       -> FETCH_HEAD
2021-10-04T18:52:01.9391125Z  * [new branch]          main       -> upstream/main
2021-10-04T18:52:03.0673227Z From https://github.com/knative/serving
2021-10-04T18:52:03.0674297Z  * branch                main       -> FETCH_HEAD
2021-10-04T18:52:03.0769798Z Committer identity unknown
2021-10-04T18:52:03.0770617Z 
2021-10-04T18:52:03.0771194Z *** Please tell me who you are.
2021-10-04T18:52:03.0771484Z 
2021-10-04T18:52:03.0771758Z Run
2021-10-04T18:52:03.0771961Z 
2021-10-04T18:52:03.0772862Z   git config --global user.email "you@example.com"
2021-10-04T18:52:03.0773579Z   git config --global user.name "Your Name"
2021-10-04T18:52:03.0773901Z 
2021-10-04T18:52:03.0774429Z to set your account's default identity.
2021-10-04T18:52:03.0775146Z Omit --global to set the identity only in this repository.
2021-10-04T18:52:03.0775544Z 
2021-10-04T18:52:03.0780159Z fatal: empty ident name (for <runner@fv-az127-662.h3uyzbqov44edfyrgkjongjtha.bx.internal.cloudapp.net>) not allowed
2021-10-04T18:52:03.0791740Z ##[error]Process completed with exit code 128.
2021-10-04T18:52:03.0905884Z Post job cleanup.
2021-10-04T18:52:03.1870293Z [command]/usr/bin/git version
2021-10-04T18:52:03.1909015Z git version 2.33.0
2021-10-04T18:52:03.1940141Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2021-10-04T18:52:03.1971539Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2021-10-04T18:52:03.2236768Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2021-10-04T18:52:03.2253378Z http.https://github.com/.extraheader
2021-10-04T18:52:03.2269080Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2021-10-04T18:52:03.2292750Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2021-10-04T18:52:03.2587439Z Cleaning up orphan processes
```

--- a/workflow-templates/knative-go-test.yaml
+++ b/workflow-templates/knative-go-test.yaml
@@ -52,6 +52,12 @@ jobs:
       - name: Merge upstream
         if: github.event_name == 'pull_request'
         run: |
+          if ! git config user.name > /dev/null; then
+            git config user.name "John Doe"
+          fi
+          if ! git config user.email > /dev/null; then
+            git config user.email "johndoe@localhost"
+          fi
           git remote add upstream https://github.com/${{ github.repository }}.git
           git fetch upstream ${{ github.base_ref }}
           git pull --no-rebase --no-commit upstream ${{ github.base_ref }}

